src/test/java/core/basesyntax/MainAppTest.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainAppTest {
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[3];
+        machines[0] = new Truck();
+        machines[1] = new Bulldozer();
+        machines[2] = new Excavator();
+
```

Comment:
remove redundant empty line

Reply:
I tried . it does not let me remove them in intelllij idea